src/components/SequenceInput/SequenceInput.js

```diff
@@ -26,13 +26,16 @@ export default function SequenceInput({
   handleOnUpdateCard,
   onUpdateCancel,
   isEmpty,
+  sceneSelected,
 }) {
   const tags = ['Tor', 'Rote Karte']
   const history = useHistory()
 
   const [activeTagIndex, setActiveTagIndex] = useState(null)
```

Comment:
Jerry recommended to combine several smaller states into a larger one to increase readability and performance: Pot. group up description, timecode and playerName if that works within the architecture of your app